specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2024-03-01/virtualMachine.json

```diff
@@ -3337,6 +3337,10 @@
           },
           "description": "The virtual machine zones."
         },
+        "placement": {
+          "$ref": "#/definitions/Placement",
+          "description": "The virtual machine automatic zone placement feature."
```

Comment:
Placement section specifies the user-defined constraints for virtual machine placement on hardware. Minimum api-version: 2024-03-01.